recipes/itolapi/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "itolapi" %}
+{% set version = "4.0.0" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 161759384603664ff21ad6ae412a01549ce19436fc6959478d932d9b6c9915c4
+
+build:
+  noarch: generic
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
+
+requirements:
+  host:
+    - python<3.8
```

Comment:
I think the error message in the log is misleading. As far as I can tell, the requirement should actually be `python >=3.6`.

Reply:
indeed, this works locally

Reply:
If you use `noarch: python`, the package is going to be version-independent anyway. And Python packages that are version-specific are built only for 3.6 and 3.7 at the moment in Bioconda, so so the restriction doesn’t do anything.

Reply:
you mean **noarch: python** I guess ?

Reply:
okay

Reply:
You should use `noarch: python` here. I’m not entirely sure what the difference is to `noarch: generic`, but [the docs want `noarch: python` for pure-Python packages](https://bioconda.github.io/contributor/guidelines.html?highlight=noarch#noarch-packages)

Reply:
(Perhaps I misunderstood your question; let me know if I did.)